view/frontend/web/template/payment/form.html

```diff
@@ -20,49 +20,59 @@
             <!-- ko template: getTemplate() --><!-- /ko -->
             <!--/ko-->
         </div>
+
         <form id="form-billink" class="form" data-mage-init='{"validation": {}}' method="post" action="#">
             <fieldset data-bind="attr: {class: 'fieldset ' + getCode(), id: 'payment_form_' + getCode()}">
                 <legend class="legend">
                     <span><!-- ko i18n: 'Billink Billing Information'--><!-- /ko --></span>
                 </legend>
                 <br/>
 
-                <div class="field radio required">
-                    <span class="label"><!-- ko i18n: 'Workflow Type'--><!-- /ko --></span>
-
-                    <div class="control">
-                        <!-- ko foreach: { data: customerTypes, as: 'workflow' } -->
-                        <input type="radio" name="payment[customer_type]" class="input-text"
-                               data-validate="{'validate-one-required-by-name':true}"
-                               data-bind="disable: !$parent.showWorkflowOption(workflow.value),
-                                    value: workflow.value,
-                                    checked: $parent.selectedCustomerType,
-                                    attr: {
-                                        autocomplete: 'off',
-                                        id: $parent.getCode() + '_customer_type_' + workflow.type.toLowerCase(),
-                                        title: $t('Customer Type ' + workflow.type)}"/>
-                        <label data-bind="attr: {for: $parent.getCode() + '_customer_type_' + workflow.type.toLowerCase()}"
-                               class="label">
-                            <span data-bind="text: workflow.type"></span>
-                        </label>
-                        <br/>
-                        <!--/ko-->
+                <div class="field NEW">
```

Comment:
@patrickcoppens Can't find a reference to class `NEW`. Looks like a debug leftover?

Reply:
Done